### PR TITLE
Make the boot.iso smaller

### PR DESCRIPTION
**Pull request by bcl**

### CODE REVIEW

**Comment by jwboyer** [comment]
We should hold off on merging the wireless change.  That's not confirmed yet.

File: share/runtime-cleanup.tmpl [hunk 1]
```diff
@@ -344,6 +344,9 @@ removefrom redhat-logos /usr/share/{plymouth,backgrounds}*
 removefrom subscription-manager --allbut /etc/rhsm/* /usr/share/rhsm/* /var/log/rhsm/* \
     /etc/yum/pluginconf.d/product-id.conf /usr/lib/yum-plugins/product-id.py
 
+## OSCAP cleanups
+removefrom scap-security-guide --allbut /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
+
```
Comment (jwboyer) [inline share/runtime-cleanup.tmpl]: Could add:

removefrom scap-security-guide /usr/share/scap-security-guide/

as well.

Comment (bcl) [reply share/runtime-cleanup.tmpl]: That already removes those files as well. --allbut removes every file in the packages except those matching the argument glob.

**Comment by bcl** [comment]
Closing this one, the oscap files were removed but the firmware left as is.

**Pull request closed without merging**